projects/breast_cancer/README.md

```diff
@@ -0,0 +1,137 @@
+
+References:  
+[1] http://emedicine.medscape.com/article/1947145-overview#a3  
+[2] http://emedicine.medscape.com/article/1947145-overview#a7  
+[3] http://emedicine.medscape.com/article/1954658-overview  
+[4] http://emedicine.medscape.com/article/1947145-workup#c12  
+
+## Goal & Approach
+In an effort to automate the process of classification, this project aims to develop a large-scale deep learning approach for predicting tumor scores directly from the pixels of whole-slide histopathology images.  Our proposed approach is based on a recent research paper from Stanford [1].  Starting with 500 extremely high-resolution tumor slide images with accompanying score labels, we aim to make use of Apache Spark in a preprocessing step to cut and filter the images into smaller square samples, generating 4.7 million samples for a total of ~7TB of data [2].  We then utilize Apache SystemML on top of Spark to develop and train a custom, large-scale, deep convolutional neural network on these samples, making use of the familiar linear algebra syntax and automatically-distributed execution of SystemML [3].  Our model takes as input the pixel values of the individual samples, and is trained to predict the correct tumor score classification for each one.  In addition to distributed linear algebra, we aim to exploit task-parallelism via parallel for-loops for hyperparameter optimization, as well as hardware acceleration for faster training via a GPU-backed runtime.  Ultimately, we aim to develop a model that is sufficiently stronger than existing approaches for the task of breast cancer tumor proliferation score classification.
+
+References:  
+[1] https://web.stanford.edu/group/rubinlab/pubs/2243353.pdf  
+[2] See [`Preprocessing.ipynb`](Preprocessing.ipynb).  
+[3] See [`MachineLearning.ipynb`](MachineLearning.ipynb), [`softmax_clf.dml`](softmax_clf.dml), and [`convnet.dml`](convnet.dml).  
+
+![Approach](https://dusenberrymw.github.io/incubator-systemml/img/projects/breast_cancer/approach.svg)
```

Comment:
Yeah, I'll update that when I merge.  Wanted to make sure the image showed up in the PR still as it requires that it be hosted on the gh-pages branch in order to reference it from the README.